### DISCUSSION TITLE
Download speed is too slow

### DISCUSSION THREAD

**Discussion by KooroshRH** [bdd100k/bdd100k#287] [Q&A]
Hi,

I hope you are doing well. I want to download the primary 100k dataset in parts, but the download speed from your servers is too slow. It would be great to have their links from google drive or fix the issue with the servers.

Thanks

**Comment by lone-duck** [comment]
Did you ever find a solution to this?

**Reply by KooroshRH** [reply]
Unfortunately not yet.

**Reply by zyandtom** [reply]
hi if you have downloaded all videos, could you please share them?
